conf/bridge-server.conf

```diff
@@ -3,6 +3,7 @@ synapse.api.key = your-api-key-here
 
 exporter.synapse.id = 3325672
 test.synapse.user.id = 3348228
+admin.synapse.user.id = 3351155
```

Comment:
I thought we decided on having a list of individual accounts rather than having a shared account?